### ISSUE TITLE
Make a demo of the app and put it as a gif in the README.md

### ISSUE THREAD

**Issue by gargantuadev** [pooja-gera/algorithm-finder#14]
Hey @pooja-gera, I would like to improve the login UI. May you kindly assign to me this issue?

I would like to add some dynamism to it, thank you!

**Comment by pooja-gera** [comment]
Hey @gargantuadev this issue would be low priority as of now, however there's a small non-technical task if you want to contribute there. 

We need to update `README.md` with the latest demo of the app ^.^ 

I follow this flow but you can use whatever is comfortable:
- Screen record a short clip (I use mac's inbuilt tool for screen recording, you can use OBS Studio / Windows' Game Center if you are on a Windows machine)
- Convert the clip from `.mov` / `.mp4` to `.gif`
- Upload the GIF to cloudinary 
- Copy the public URL and paste it in the `README.md` file 

Feel free to edit this issue only with the updated requirements and open a PR for it.

**Comment by gargantuadev** [comment]
Hey there @pooja-gera!

I was going to do the task about recording, but I noticed with the latest PRs the correct behaviour has changed...
![image](https://github.com/user-attachments/assets/cb1e30bb-0dfd-4175-b207-e3d059115536)

Basically the visitor count has been moved to the left (only the title) and when you open the app is not asking you anymore to log, but they just moved the button at the top - right, is this expected? Before when you were starting the app it was not letting you use it until you were logged in. Do you want to fix this? before doing the other task on recording?

**Comment by pooja-gera** [comment]
Hello @gargantuadev I was waiting for the recent PR to be merged, the app is now responsive as well so you won’t see this behaviour. 

And, yes the login behavior has changed, on a second thought I had some concerns over putting auth completely on FE so removed the guardrail of people accessing the app only after logging in. But to preserve your efforts I put a button on the side in case people want to try and experiment on local. 

I have plans in the future to create a separate auth-service and handle login from there.

**Comment by gargantuadev** [comment]
Hello @pooja-gera, no worries then, I'll do the PR about the README.md update!

**Comment by pooja-gera** [comment]
Lovely. Please edit the title of issue and link it to your PR 🤩

**Comment by gargantuadev** [comment]
Hello there @pooja-gera!
I've been pretty busy, but I managed to find the time to do it 😝.

The pr is the #20, don't hesitate to let me know if everything is as expected ;)
